src/Microsoft.ManagedZlib/benchmarks/CompressFiles.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.IO;
+using System.IO.Compression;
+using System.Text;
+
+namespace Microsoft.ManagedZLib.Benchmarks
+{
+    class CompressFiles
+    {
+        
+        private string zipName = @".\resultZip.zip";
```

Comment:
IDE0044: Add readonly modifier:
```suggestion
        private readonly string zipName = @".\resultZip.zip";
```